hokusai/staging.yml

```diff
@@ -190,3 +190,34 @@ spec:
                         operator: In
                         values:
                           - background
+---
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: horizon-refresh-project-data-cron
+spec:
+  schedule: "0 0 * * 1"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: horizon-refresh-project-data-cron
```

Comment:
"Data" is a little general-sounding. I might call this task and the cron `refresh_dependencies` or `refresh_components` if you want it to imply more than dependencies (like CI).